lib/plug/conn.ex

```diff
@@ -64,6 +64,9 @@ defmodule Plug.Conn do
   or elsewhere).
   More can be stored in a session cookie, but be careful: this makes requests
   and responses heavier, and clients may reject cookies beyond a certain size.
+  It can also cause bugs; for example, if the contents of a shopping cart were
+  stored in a session cookie, a user switching between browsers would see an
+  inconsistent cart.
```

Comment:
Maybe we should rephrase this to say that whatever is in the session is not shared across different browsers and devices? So you need to consider if that does not go to a database instead.

Reply:
Yep, will do

Reply:
Rephrased